keras_cv/layers/preprocessing/random_hue.py

```diff
@@ -53,12 +49,11 @@ class RandomHue(VectorizedBaseImageAugmentationLayer):
     ```
     """
 
-    def __init__(self, factor, value_range, seed=None, **kwargs):
```

Comment:
Any reason to remove the value range here?  We use it in clipping,  so I believe it is valid.

Reply:
Because Hue channel is always at [0, 1]

We can validate this from https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/kernels/image/colorspace_op.h#L49-L57

For example, we have `value_range` in [a, b] indicates a is the min and b is the max
The value range of:
- `V`: [a, b]
- `range` : [0, b - a]
- `norm`: [0, (b-a)*(1/6)]
So the min of `H` is `0` if `R==V` and `G-B=0`
and the max of `H` occurs at `R==V` and `G-B` is slightly negative => value will be `H=H+1` at line57 and becomes very close to `1`

There is a colab to verify the behavior of this
https://colab.research.google.com/drive/1KBPQCL9pxI4Ar6KvnhfzncbTI8zT6kGq?usp=sharing

```bash
value_range: [0, 1]
  H channel: max= 1.00000;min= 0.00000
value_range: [0, 255]
  H channel: max= 1.00000;min= 0.00000
value_range: [0, 100]
  H channel: max= 1.00000;min= 0.00000
value_range: [-50, 100]
  H channel: max= 1.00000;min= 0.00000
```

In RandomHue, we add `adjust_factors` which is in range [-0.5, 0.5] and post-process by
```python
h_channel = tf.where(h_channel > 1.0, h_channel - 1.0, h_channel)
h_channel = tf.where(h_channel < 0.0, h_channel + 1.0, h_channel)
```
So the H channel is always at [0, 1] no matter what we do with `tf.image.rgb_to_hsv` and the operations of RandomHue

Reply:
@LukeWood 
There is no `value_range` in RandomSaturation for the same reason.
https://github.com/keras-team/keras-cv/pull/1411/files#r1112453229

To be more specific, the `value_range` in previous RandomHue is not for clipping ([previous implementation](https://github.com/keras-team/keras-cv/blob/15056344d08d234537fdf5b61d77c66c1fae691b/keras_cv/layers/preprocessing/random_hue.py#L81-L101)), but for value transformation (which is useless)